framework/decode/vulkan_cpp_consumer_base.cpp

```diff
@@ -2285,6 +2303,18 @@ void VulkanCppConsumerBase::GenerateDescriptorUpdateTemplateData(DescriptorUpdat
                 variables.push_back(std::move(offset));
                 break;
             }
+            case VK_DESCRIPTOR_TYPE_INLINE_UNIFORM_BLOCK:
+            {
+                // TODO: needs testing, unsure when/how we'll use this
```

Comment:
Could you add a issue for this TODO? The others look good to me.

Reply:
done -> https://github.com/LunarG/gfxreconstruct/issues/1549